cypress/integration/artist.spec.js

```diff
@@ -20,6 +20,5 @@ describe("/artist/:id", () => {
     cy.get("h1").should("contain", "Pablo Picasso")
     cy.get("h2").should("contain", "Spanish, 1881–1973")
     cy.get("h2").should("contain", "Notable Works")
-    cy.get("h2").should("contain", "Iconic Collections")
```

Comment:
Rail is lazy loaded via scroll interception on the client, and far down the page